### ISSUE TITLE
Not getting description for function argument

### ISSUE THREAD

**Issue by PMassicotte** [R-nvim/R.nvim#87]
I am not getting any information/description on function argument. I am missing something?

![Peek 2024-03-17 06-52](https://github.com/R-nvim/R.nvim/assets/4519221/71d3dac0-f909-4bfe-ba91-077571abb571)

**Comment by jalvesaq** [comment]
I get most arguments for `lm`. I don't get the description of `x`, `y`, and `model`, but I get the description of `qr`. So, the bug on my side is the need to copy the description when two or more arguments are described together.

**Comment by PMassicotte** [comment]
I am not getting description for any functions.

**Comment by jalvesaq** [comment]
The descriptions are saved in the `args_` files at `~/.cache/R.nvim`. In this case, `args_stats_`, and you should find a line beginning with `lm^F` within it, where `^F` is the byte `x06`.

**Comment by PMassicotte** [comment]
This is what I have:

![image](https://github.com/R-nvim/R.nvim/assets/4519221/68a6f869-2a35-4bfa-8d97-fec5e9082708)

Does it look ok?

**Comment by jalvesaq** [comment]
I can see `^F` instead of empty space:

![image](https://github.com/R-nvim/R.nvim/assets/891655/c732e75f-a732-470e-aba0-25b117f8b71c)

**Comment by jalvesaq** [comment]
`^F` (`\x06`) is used to separate arguments, and `^E` (`\x05`) is used to separate arguments from their descriptions.

**Comment by PMassicotte** [comment]
I am a bit lost now :) I understand that _section_ are not correctly separated on my side?

**Comment by PMassicotte** [comment]
There are present, if you zoom on my image, just very pale

**Comment by jalvesaq** [comment]
So, the problem isn't there...

**Comment by jalvesaq** [comment]
If I put `options(nvimcom.verbose=3)` in my `~/.Rprofile`, I can see this in the R Console after opening the parenthesis of `lm`:

```
nvimcom Received: [E] nvimcom:::nvim_complete_args("2", "lm", "")
send_to_nvim [5] {2553555838}: +A2;;lm;#
```

The message `+A2;;lm;#` sent from `nvimcom` to `rnvimserver` means that `R.nvim` should look for arguments for `lm` and not for something else (a method of another function).

**Comment by PMassicotte** [comment]
Ok it works now. I am using [chezmoi](https://www.chezmoi.io/) to manage my dotfiles and somehow, I had an older version of cmp-r which pointed to an older branch. All good now and sorry for that!

**Comment by jalvesaq** [comment]
Good to know that it wasn't an R.nvim bug, and that you managed to fix it!

**Comment by jalvesaq** [comment]
> I get most arguments for `lm`. I don't get the description of `x`, `y`, and `model`, but I get the description of `qr`. So, the bug on my side is the need to copy the description when two or more arguments are described together.

This bug was fixed by https://github.com/R-nvim/R.nvim/commit/268832c8c26bb966ef485dbe58a4d3177a0a85a0

**Comment by PMassicotte** [comment]
Very nice!

**Comment by PMassicotte** [comment]
> > I get most arguments for `lm`. I don't get the description of `x`, `y`, and `model`, but I get the description of `qr`. So, the bug on my side is the need to copy the description when two or more arguments are described together.
> 
> This bug was fixed by [268832c](https://github.com/R-nvim/R.nvim/commit/268832c8c26bb966ef485dbe58a4d3177a0a85a0)

Just tested it and it works perfectly.
